org.nuspell.Nuspell.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
Ideally it should be maintained in Nuspell code repository and installed by its build system.

Reply:
We only use this for flatpak and appimage. For the latter, we generate it on the fly, see https://github.com/nuspell/misc-nuspell/blob/master/packaging/appimage/Nuspell.yml Would that also be possible for flatpak? Do you know of examples?